### PR TITLE
Off switch

### PR DESCRIPTION
**Pull request by NotSimon5673**
pull request for merging offSwitch into main

### CODE REVIEW

**Comment by KenwoodFox** [comment]
This seems to break more than it fixes ;/

**Comment by NotSimon5673** [comment]
> This seems to break more than it fixes ;/

probably because this branch is so many commits behind

**Comment by KenwoodFox** [comment]
I bumped it up with a rebase, ill see if i can fix it up at home

File: admin_bot/cogs/tools.py [hunk 1]
```diff
@@ -14,6 +14,10 @@ class ToolCog(commands.Cog, name="Tools"):
     def __init__(self, bot):
         self.bot = bot
 
+        # Random chance to rick-roll
+        if self.bot.rick():
+            return
+
```
Comment (KenwoodFox) [inline admin_bot/cogs/tools.py:20]: Prolly dosent belong here

File: admin_bot/cogs/tools.py [hunk 2]
```diff
@@ -67,35 +71,17 @@ async def feature(self, ctx, *args):
         )
 
     @commands.command()
-    async def get_help(self, ctx, team, *args):
+    @commands.has_any_role("Lead of Software", "Assistant lead of software")
+    async def kill(self, ctx):
```
Comment (KenwoodFox) [inline admin_bot/cogs/tools.py:75]: The get help command becomes the kill command? :o

File: admin_bot/cogs/tools.py [hunk 3]
```diff
@@ -67,35 +71,17 @@ async def feature(self, ctx, *args):
...
-            await ctx.send(
-                f"```asciidoc\n[HELP TICKET]\n{message}\n------------------\n```"
-            )
-
-        else:
-            await ctx.send(
-                f"Unrecagnized team {team}, please use one of the following:\nKode, Mechanical, Electrical, CAD, Mentor, all"
-            )
+        await ctx.send(str("shutting off"))
+        await ctx.bot.logout()
```
Comment (KenwoodFox) [inline admin_bot/cogs/tools.py:84]: Idk if this quits the main process, will have to test

**Review by KenwoodFox — CHANGES_REQUESTED** [review]
Ill see if i can get this running at home, may make some changes

File: admin_bot/cogs/tools.py (hunk 2 above)
Comment (NotSimon5673) [reply admin_bot/cogs/tools.py:75]: hm. I'll fix that

**Pull request closed without merging**